nixos/modules/installer/tools/nixos-version.sh

```diff
@@ -13,6 +13,13 @@ case "$1" in
     fi
     echo "@revision@"
     ;;
+  --configuration-revision)
+    if [[ "@configurationRevision@" =~ "@" ]]; then
```

Comment:
This is kind of awful, but probably fine for now. I think nixos-version would make a good candidate for `__structuredAttrs` once https://github.com/NixOS/nixpkgs/pull/175649 lands in master.

Reply:
I also did not particularly like it - I am happy to take a look into `__structuredAttrs` - should this included in this PR?